service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/transport/amqps/AmqpFileUploadNotificationReceivedHandler.java

```diff
@@ -227,39 +149,9 @@ public void onLinkInit(Event event)
         Link link = event.getLink();
         if (event.getLink().getName().equals(FILE_NOTIFICATION_RECEIVE_TAG))
         {
-
-            Target t = new Target();
```

Comment:
Why are we removing this?

Reply:
A link only needs a target if it is a sender link. Receiver links only need a source. It didn't hurt to give a sender link a source or a receiver link a target, but it would be confusing to keep that code